services/aws-s3.js

```diff
@@ -0,0 +1,113 @@
+const config = require('config');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const Sharp = require('sharp');
+
+/**
+ * AWSS3 Example of simple class with basic functionality used to upload
+ * files to Amaozn S3 bucket
+ *
+ * @author Maciej Lisowski
+ * @since 2018-11-27
```

Comment:
no.. I'm thinking services to work as basically where you put all the business logic. I've been personally moving away from a model/controller framework into more handler/services where the model is barebones, and the handler or service contains all the business logic. 

I'm ok with moving back to a more traditional architecture though... The one thing is that with mongo schema's i'm not sure how that will look.

Reply:
meh, i'm okay either way, just looking to understand how to work it. If it works better with mongo (because its relatively schema-less?) I'm cool with handler/services approach.

Reply:
well, think about it and then tell me.

Reply:
sure lets try for handler/services approach. Worst case we refactor or start from scratch. Can't be too hard to do that on work between the two of us.